doc/plugins/limit-req.md

```diff
@@ -21,11 +21,14 @@
 
 # Summary
 
-- [**Name**](#name)
-- [**Attributes**](#attributes)
-- [**How To Enable**](#how-to-enable)
-- [**Test Plugin**](#test-plugin)
-- [**Disable Plugin**](#disable-plugin)
+- [Summary](#summary)
+  - [Name](#name)
+  - [Attributes](#attributes)
+  - [How To Enable](#how-to-enable)
+  - [Test Plugin](#test-plugin)
+    - [How to enable on the `consumer`](#how-to-enable-on-the-consumer)
+    - [Test Plugin](#test-plugin-1)
```

Comment:
[Test Plugin]()
[How to enable on the `consumer`]()

I think this sorting is better.

Reply:
Important chapters should be first.

Reply:
fixed.